js/about/preferences.js

```diff
@@ -85,224 +83,6 @@ const braveryPermissionNames = {
   'noScript': ['boolean', 'number']
 }
 
-class BitcoinDashboard extends ImmutableComponent {
```

Comment:
https://github.com/brave/browser-laptop/pull/7481/commits/f3754515ccab859ee56acd09d168c95c722cc871 it's already in